### PR TITLE
Fixes #36590 - fix ACS error message for duplicate words

### PR DESCRIPTION
**Pull request by lfu**
#### What are the testing steps for this pull request?
1. Add new simplified ACS
2. Have an active product that does not have a file repository and try to add it to a new ACS (i.e. RHEL)

### CODE REVIEW

File: app/lib/katello/validators/alternate_content_source_products_validator.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ def validate_each(record, attribute, value)
             product = ::Katello::Product.find(value)
             content_type = record.alternate_content_source.content_type
             if product.repositories.with_type(content_type).has_url.empty?
-              record.errors[attribute] << N_("The product %{name} has no %{type} repositories with upstream URLs to add to the alternate content source.") % { name: product.name, type: content_type }
+              record.errors.add(attribute, N_("%{name} has no %{type} repositories with upstream URLs to add to the alternate content source.") % { name: product.name, type: content_type })
```
Comment (lfu) [inline app/lib/katello/validators/alternate_content_source_products_validator.rb:10]: Do you think this should be `N_` or `_`?

Comment (sjha4) [reply app/lib/katello/validators/alternate_content_source_products_validator.rb:10]: Not entirely sure, but documentation suggests N_ for dynamic translations..
https://github.com/grosser/gettext_i18n_rails#unfound-translations-with-rake-gettextfind

Comment (lfu) [reply app/lib/katello/validators/alternate_content_source_products_validator.rb:10]: Just tested it out. 

With `N_`
Screenshot at 2023-07-13 17-11-28

With `_`
Screenshot at 2023-07-13 17-14-12

**Review by sjha4 — APPROVED** [review]
Ack. Can see updated error message:
Before chnage:
Screenshot from 2023-07-14 16-40-52
---
After change:
Screenshot from 2023-07-14 16-38-08

**Pull request merged**